app/models/google_calendar/event_editor.rb

```diff
@@ -0,0 +1,44 @@
+module GoogleCalendar
+  class EventEditor
+    def initialize(credentials)
+      @credentials = credentials
+      @calendar_service = GoogleCalendar::Client.new(credentials).calendar_service
+    end
+
+    def finish(conference_room_id, event_id)
+      @conference_room_email = ConferenceRoom.find(conference_room_id).email
+      @event = get_event(event_id)
+      finish_event
+    end
+
+    private
+
+    def get_event(event_id)
+      calendar_service.get_event(conference_room_email, event_id)
+    end
+
+    def update_event
+      calendar_service.update_event(conference_room_email, event.id, event)
+    end
+
+    def finish_event
+      return if event_not_started? || event_finished?
```

Comment:
Have you considered how would it behave when two users hit this method for the same event at the same time simultaneously?